lib/core/src/model.rs

```diff
@@ -698,10 +698,11 @@ pub struct PrepareSendRequest {
     /// The destination we intend to pay to.
     /// Supports BIP21 URIs, BOLT11 invoices, BOLT12 offers and Liquid addresses
     pub destination: String,
-
     /// Should only be set when paying directly onchain or to a BIP21 URI
     /// where no amount is specified, or when the caller wishes to drain
     pub amount: Option<PayAmount>,
+    /// An optional comment for this payment
```

Comment:
Nit: IMO it would be helpful to document that this may only apply to bolt12 destinations

Reply:
Thanks, fixed [49efa5c](https://github.com/breez/breez-sdk-liquid/pull/930/commits/49efa5c6d1c0d88b5eded64468cd7902212676b2)